lib/product.rb

```diff
@@ -0,0 +1,22 @@
+class Product
+  attr_reader :category, :name, :unit_price, :quantity, :hoard
+  def initialize(category, name, unit_price, quantity)
+    @category = category
+    @name = name
+    @unit_price = unit_price
+    @quantity = quantity.to_i
+    @total_price = total_price
+
+  end
+
+  def total_price
+    unit_price * quantity
+  end
+
+  def is_hoarded?
+    false
+    if hoard
+      true
+    end
+  end
```

Comment:
Make sure you are continually running your tests - this method is not actually passing 😬